### PR TITLE
GH-639 Align ModelState API with Java GLSP Server API

### PR DESCRIPTION
**Pull request by tortmayr**
Introduce `updateRoot` method and make root setter protected
- Update changelog
Fixes https://github.com/eclipse-glsp/glsp/issues/639

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
 This project provides the Node based server component for the Eclipse Graphical Language Platform (GLSP).
 The implementation of this server is aligned with the default Java based [GLSP Server](https://github.com/eclipse-glsp/glsp-server).
-The initial initial implementation was contributed on behalf of STMicroelectronics.
+The [initial implementation](https://github.com/eclipse-glsp/glsp-server-node/commit/4fba8e8beef07798a7eff27c9c04ca68583e5960) has been contributed on behalf of STMicroelectronics.
+The following list composes changes that have been made since the initial implementation.
 
 ### Changes
 
--   [diagram] Implement LayoutEngine API for server-side autolayouting & provide an integration package for layout engines based on ELK. [#509](https://github.com/eclipse-glsp/glsp-server-node/pull/2) [#514](https://github.com/eclipse-glsp/glsp-server-node/pull/5) - Contributed on behalf of STMicroelectronics
+-   [core] Implement `dispatchOnNextUpdate` method that enables queuing of actions that should be dispatched after the next graphical model update. [#1](https://github.com/eclipse-glsp/glsp-server-node/pull/1)
```
Comment (sgraband) [inline CHANGELOG.md:13]: This was also Contributed on behalf of STMicroelectronics

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
 # Eclipse GLSP Server Node Changelog
 
-## v0.10.0- Upcoming
+## v1.0.0- Upcoming
 
 Inception of the Node GLSP Server.
 This project provides the Node based server component for the Eclipse Graphical Language Platform (GLSP).
 The implementation of this server is aligned with the default Java based [GLSP Server](https://github.com/eclipse-glsp/glsp-server).
-The initial initial implementation was contributed on behalf of STMicroelectronics.
+The [initial implementation](https://github.com/eclipse-glsp/glsp-server-node/commit/4fba8e8beef07798a7eff27c9c04ca68583e5960) has been contributed on behalf of STMicroelectronics.
```
Comment (sgraband) [inline CHANGELOG.md:8]:
```suggestion
The [initial implementation](https://github.com/eclipse-glsp/glsp-server-node/commit/4fba8e8beef07798a7eff27c9c04ca68583e5960) was contributed on behalf of STMicroelectronics.
```
Nitpick, but i think the process is over, hence `was` is more suited than `has been`.

**Review by sgraband — COMMENTED** [review]
Thanks @tortmayr! Changes look good overall. Just two small remarks!

**Comment by tortmayr** [comment]
Thanks for the fast review Simon. I pushed an update with the requested changes.

**Review by sgraband — APPROVED** [review]
Thank you for the changes! LGTM :+1:

**Pull request merged**